register.html

```diff
@@ -20,15 +20,26 @@ <h1 class="logo">Relaxr</h1>
 					<ul>
 						<li><a href="index.html">Home</a></li>
 						<li><a href="blog.html">Blog</a></li>
-						<li><a href="#">Contact</a></li>
+						<li><a href="register.html">Contact</a></li>
 					</ul>
 				</nav>
 			</div>
 		</header>
 		<section id="main">
 			<div class="container">
+				<h2 class="register">Create an account</h2>
+				<div id="errorMessage" class="error hide"></div>
+				<div id="errorMessage2" class="error hide"></div>
+				<form id="registrationForm">
+					<label for="username">Username</label>
+					<input type="text" name="username" placeholder="Username">
+					<label for="username">Password</label>
+					<input type="password" name="password" placeholder="Password">
+					<label for="username">Email</label>
+					<input type="email" name="emailAddress" placeholder="Email">
+				</form>
 				<section id="contact">
-					<input type="submit" value="Register Now!">
+					<input class="buttonRegister" type="submit" value="Register Now!">
```

Comment:
This element should be part of the form - see above comment. The JavaScript won't work without it being inside of the form.